### PR TITLE
Fix inittialization order issue.

### PR DESCRIPTION
**Pull request by jplevyak**
Signed-off-by: John Plevyak <jplevyak@gmail.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

**Comment by PiotrSikora** [comment]
Why are you removing examples? If it's because of `check_spelling`, then you can exclude `examples/wasm/envoy_filter_http_wasm_example.wat` by adding it to `tools/spelling_skip_files.txt`.

**Comment by jplevyak** [comment]
re-added those files and added them to tools/spelling_skip_files.txt

File: tools/spelling_skip_files.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 OWNERS.md
 test/extensions/filters/http/wasm/test_data/metadata_cpp.wat
+examples/wasm/envoy_filter_http_wasm_example.wat
+examples/wasm/envoy_filter_http_wasm_example.wasm
```
Comment (PiotrSikora) [inline tools/spelling_skip_files.txt]: Nit: `check_spelling` complains only about the `.wat` file.

Side-note: we could probably just use a `*.wat` regex here to future-proof it.

Comment (jplevyak) [reply tools/spelling_skip_files.txt]: Done.

**Pull request merged**